Extensions/TileMap/JsExtension.js

```diff
@@ -436,6 +486,13 @@ module.exports = {
       if (this._pixiObject.layerIndex !== layerIndex)
         this._pixiObject.layerIndex = layerIndex;
 
+      const animationSpeed = this._associatedObject
+        .getProperties(this.project)
+        .get('animationSpeed')
+        .getValue();
+      if (this._pixiObject.animationSpeed !== animationSpeed)
+        this._pixiObject.animationSpeed = animationSpeed;
```

Comment:
Please be careful about setting properties like you're doing here on a pixiObject. We already had a huge bug that took us hours of debugging because `render` or something like this was set. 🙏🙏

Is animationSpeed something that is used by the _pixiObject, i.e: by the tilemap. If no, then don't set it! You have `this` if you want to store something, but please don't add arbitrary property to objects that are not made for this.
Anyway we don't even need this because we're not using it in the editor :)

Reply:
it is used by the extension yes, but it is not a part of pixi-tilemap.
This is is similar to how some other extensions work. Are you suggesting to attach these variables elsewhere - not to pixiObject? I do need them all - they do stuff as of now too - if you remove them parts of the extension will stop working

Reply:
> Are you suggesting to attach these variables elsewhere - not to pixiObject

Yes, in `this`, not in `this._pixiObject`. 

> how fast do we increment frames (animationSpeed)

We don't use that in the editor, so you can entirely remove `this._pixiObject.animationSpeed = animationSpeed` - it's not used anywhere in the editor unless I missed something.

Remember: the pixi object is not a place where to store variables. There is `this` (i.e: the `RenderedTileMapInstance`) for storing whatever you need to render your instance in the editor.